### PR TITLE
feat(control-interface): Updates control interface docs

### PR DESCRIPTION
**Pull request by thomastaylor312**
This adopts a similar style to the docs for the wadm API and updates things to explain the latest updates to actions like scaling.

### CODE REVIEW

File: docs/hosts/lattice-protocols/control-interface.md [hunk 1]
```diff
@@ -1,50 +1,591 @@
...
+group with the annotations `{ "foo": "bar" }`.
+
+#### Ping Hosts
+
+`nats req wasmbus.ctl.{lattice_prefix}.ping.hosts ''`
+
+Pings all hosts in the lattice and gathers responses. This is a "scatter/gather" type operation,
+meaning that you'll receive multiple json responses from all hosts until your configured timeout. If
+using a nats client (like the NATS CLI), you'll need to set `--replies 0 --timeout <your timeout>`.
+An example response body can be found below.
```
Comment (connorsmith256) [inline docs/hosts/lattice-protocols/control-interface.md]: Nit: this feels unnecessary, since we've been including example responses on everything

**Review by connorsmith256 — CHANGES_REQUESTED** [review]
This is incredible ❤️ 🚀 🙌 🎉 

Just one nit, and one request:

For a lot of the responses you show an object that includes a `message` field, but I don't see that in the host's implementation. Are these copy-pasta mistakes? https://github.com/wasmCloud/wasmCloud/blob/main/crates/host/src/wasmbus/mod.rs#L70

File: docs/hosts/lattice-protocols/control-interface.md (hunk 1 above)
Comment (thomastaylor312) [reply docs/hosts/lattice-protocols/control-interface.md]: Good point, I'll remove

**Comment by thomastaylor312** [comment]
Turns out it was `error` and not `message` so that's fixed now

**Review by connorsmith256 — APPROVED** [review]

**Pull request merged**